app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+         
+  has_many :stories
+  has_many :collections, dependent: :destroy
+  
+  def collection_for(story)
+    collections.where(story_id: story.id).first
```

Comment:
this is kind of a weird name for a method when i read it outloud.

and not a good idea to do .first on something

what are u thinking with this one? what's it for?